src/System.Private.ServiceModel/tools/IISHostedWcfService/App_code/TestHost.cs

```diff
@@ -93,9 +136,14 @@ public byte[] GetPeerCert(bool exportAsPem)
             }
         }
 
-        public string GetFQDN()
+        public Stream Ping()
```

Comment:
I'm fine with a streamed message we could echo.  You're right, the 200 is probably what we would key off anyway. Is there any useful info we could add to the message, such as the machine it is running on?  No changes required for this PR -- just questions.